src/mantine-core/src/Hr/Hr.styles.ts

```diff
@@ -26,6 +28,23 @@ export default createUseStyles(
       borderTopStyle: variant,
       margin: 0,
     }),
+    hrVertical: ({ theme, size, variant, color }: HrStylesProps) => ({
+      border: 0,
+      borderLeftWidth: getSizeValue({ size, sizes }),
+      borderLeftColor: getThemeColor({
+        theme,
+        color,
+        shade: theme.colorScheme === 'dark' ? 4 : 6,
+      }),
+      borderLeftStyle: variant,
+    }),
+    subHeader: ({ theme, color, orientation, subHeaderProps }: HrStylesProps) => ({
+      fontSize: theme.fontSizes.xs,
+      color: subHeaderProps?.color
```

Comment:
styles should not work with `subHeaderProps`, these values are already handled in Text component, this entire class should be removed

Reply:
```tsx
<Text color={color} {...subHeaderProps} className={cx(subHeaderProps?.className)}>
  {subHeader}
</Text>
```
So finally, this would do fine?